docs/pages/enroll-resources/database-access/enroll-aws-databases/aws-docdb.mdx

```diff
@@ -0,0 +1,235 @@
+In this guide, we will use the example "teleport-docdb-user" for this role.
+
+Now click **Add new tag** at Step 3, use `TeleportDatabaseService` for the key
+and `Allowed` for the value. Then click **Create Role** to complete the process.
+
+![Create Role Step 3](../../../../img/database-access/guides/aws-create-role-add-tags.png)
+
+### Create a DocumentDB user
+
+Login your DocumentDB cluster with your master username and password from a
+machine that has network access to the DocumentDB cluster. Create a DocumentDB
+user with the IAM role ARN and use the `MONGODB-AWS` mechanisms for
+authentication:
+```code
+use $external;
+db.createUser(
+    {
+        user: "<Var name="arn:aws:iam::(=aws.aws_account_id=):role/teleport-docdb-user"/>",
+        mechanisms: ["MONGODB-AWS"],
+        roles: [ { role: "root", db: "admin" },  ]
+    }
+);
+```
+
+This example creates a DocumentDB IAM user with root privileges. You can adjust
+the privileges as required.
+
+<Admonition type="tip" title="Avoid password authentication">
+This step is a one-time setup that must use password authentication. However,
+after creating the first IAM user, you can skip password authentication by
+accessing DocumentDB through Teleport and creating more DocumentDB IAM users
+using the first IAM user.
+</Admonition>
+
+## Step 6/6 Connect
+
+Once the Database Service has started and joined the cluster, log in to see the
+registered databases:
+
+```code
+$ tsh login --proxy=<Var name="example.teleport.sh:443" /> --user=alice
+$ tsh db ls
+# Name     Description Labels
+# -------- ----------- --------
+# my-docdb             env=dev
+```
+
+Retrieve credentials for the database and connect to it as the `teleport-docdb-user` user:
+
+```code
+$ tsh db connect --db-user=teleport-docdb-user --db-name=test my-docdb
+```
+
+Log out of the database and remove credentials:
+
+```code
+$ tsh db logout rds-example
+```
+
+## Troubleshooting
+
+(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
+
+### Unable to authenticate using mechanism "MONGODB-AWS" error
+
+You may encounter the following error when IAM authentication fails:
+```code
+$ tsh db connect --db-user=teleport-docdb-user --db-name=test my-docdb
+...
+MongoServerSelectionError: error connecting to the database
+	connection() error occurred during connection handshake: auth error: sasl conversation error: unable to authenticate using mechanism "MONGODB-AWS":  Authentication failed.
+ERROR: exit status 1
+```
+
+Make sure the IAM role exists as a DocumentDB user in the "$external" database
+of the target DocumentDB cluster and the DocumentDB user is configured with
+`MONGODB-AWS` mechanisms for authentication.
+
+See [Authentication using IAM
+identity](https://docs.aws.amazon.com/documentdb/latest/developerguide/iam-identity-auth.html)
+for more details.
+
+## Next steps
+
+(!docs/pages/includes/database-access/guides-next-steps.mdx!)
+- Learn more about [Authentication using IAM identity for
```

Comment:
generally, includes should always have whitespace before and after them.
It seems to be fine here, but sometimes it causes rendering issues.

```suggestion
(!docs/pages/includes/database-access/guides-next-steps.mdx!)

- Learn more about [Authentication using IAM identity for
```